src/runtime/ui-multiplexer-particle.ts

```diff
@@ -150,12 +150,12 @@ export class UiMultiplexerParticle extends UiTransformationParticle {
         // TODO(lindner): type erasure to avoid mismatch of Store vs Handle in arc.mapHandle
         // tslint:disable-next-line: no-any
         const otherHandleStore = otherHandle.storage as any;
-        otherMappedHandles.push(`use '${await arc.mapHandle(otherHandleStore)}' as v${index}`);
+        otherMappedHandles.push(`v${index}: use '${await arc.mapHandle(otherHandleStore)}'`);
```

Comment:
nitnit - `v` prefix is a remainder of `view`, could you rename it to `h` (for handles)?